src/manifest.rs

```diff
@@ -2,14 +2,46 @@ use crate::db_state::CoreDbState;
 use crate::error::SlateDBError;
 use bytes::Bytes;
 use serde::Serialize;
+use uuid::Uuid;
 
 #[derive(Clone, Serialize, PartialEq, Debug)]
 pub(crate) struct Manifest {
+    pub(crate) parent: Option<DbLink>,
     pub(crate) core: CoreDbState,
     pub(crate) writer_epoch: u64,
     pub(crate) compactor_epoch: u64,
 }
 
+impl Manifest {
+    pub(crate) fn init_new(core: CoreDbState) -> Self {
+        Self {
+            parent: None,
+            core,
+            writer_epoch: 0,
+            compactor_epoch: 0,
+        }
+    }
+
+    pub(crate) fn init_clone(parent_db: DbLink, parent_manifest: &Manifest) -> Self {
```

Comment:
```suggestion
    pub(crate) fn new(core: CoreDbState) -> Self {
        Self {
            parent: None,
            core,
            writer_epoch: 0,
            compactor_epoch: 0,
        }
    }

    pub(crate) fn cloned(parent_db: DbLink, parent_manifest: &Manifest) -> Self {
```

Reply:
Nice suggestion. I will update the patch.